### PR TITLE
ArrayDeclaration: enforce arrow and value alignment

### PR DESCRIPTION
**Pull request by mikesherov**

### CODE REVIEW

**Comment by bmax** [comment]
👍

File: Behance/Sniffs/Arrays/ArrayDeclarationSniff.php [hunk 1]
```diff
@@ -50,7 +50,7 @@ public function process( PHP_CodeSniffer_File $phpcsFile, $stackPtr ) {
     $isShortArray = $tokens[ $stackPtr ]['code'] !== T_ARRAY;
 
     // Array keyword should be lower case.
-    if ( !$isShortArray && strtolower( $tokens[ $stackPtr ]['content'] ) !== $tokens[ $stackPtr ]['content'] ) {
+    if ( !$isShortArray && mb_strtolower( $tokens[ $stackPtr ]['content'] ) !== $tokens[ $stackPtr ]['content'] ) {
```
Comment (andyroo2000) [inline Behance/Sniffs/Arrays/ArrayDeclarationSniff.php]: Our sniffs already disallow non short-array syntax:
https://github.com/behance/php-sniffs/blob/master/Behance/Sniffs/Arrays/ShortHandArrayOnlySniff.php

File: tests/Behance/Sniffs/Arrays/ArrayDeclarationSniffTest.inc [hunk 2]
```diff
@@ -121,3 +121,15 @@ $badMultilineAssocCommaSpacing = [
...
+
+$badMultilineAssocAlignment = [
+    'a'  => 1,
+    'b' =>  2,
+    'cds' => [
+        'a'  => 1,
+        'b' =>  2,
+        'cds' =>  3
+    ]
+];
```
Comment (andyroo2000) [inline tests/Behance/Sniffs/Arrays/ArrayDeclarationSniffTest.inc]: Can we have a version of this (nested arrays) that look correct, to ensure no false positives?

File: Behance/Sniffs/Arrays/ArrayDeclarationSniff.php (hunk 1 above)
Comment (mikesherov) [inline Behance/Sniffs/Arrays/ArrayDeclarationSniff.php]: Considering this is unrelated to the change I made, perhaps file an issue?

**Comment by andyroo2000** [comment]
👍

**Comment by mikesherov** [comment]
retest
